app/build.gradle

```diff
@@ -76,14 +76,14 @@ repositories {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:support-v13:22.1.1'
-    compile 'com.android.support:appcompat-v7:22.2.1'
-    compile 'com.google.android.gms:play-services-analytics:7.5.0'
-    compile 'org.xwalk:xwalk_core_library:13.42.319.12'
+    compile 'com.android.support:support-v13:23.1.1'
+    compile 'com.android.support:appcompat-v7:23.1.1'
+    compile 'com.google.android.gms:play-services-analytics:8.3.0'
+    compile 'org.xwalk:xwalk_core_library:15.44.384.12'
 }
 
 node {
-    version = '0.12.2'
+    version = '4.2.3'
```

Comment:
This just specifies which version of Node we want the mooworks plugin to download and use to compile the app.